### ISSUE TITLE
Elements with `aria-busy` should have `aria-hidden`

### ISSUE THREAD

**Issue by robsonsobral** [robsonsobral/initial-css#21]
**[Original report](https://bitbucket.org/sobral/initial-css/issue/21) by sobral (Bitbucket: [sobral](https://bitbucket.org/sobral), GitHub: [robsonsobral](https://github.com/robsonsobral)).**

----------------------------------------

From https://developer.paciellogroup.com/blog/2018/05/short-note-on-being-busy/:

> It appears that currently only JAWS uses this information to hide busy content from the user (with Chrome and Firefox, not IE), other screen readers such as NVDA and VoiceOver ignore the busy state and the content can still be accessed.
